multigeiger/loraWan.cpp

```diff
@@ -96,59 +117,54 @@ const lmic_pinmap lmic_pins = {
     .dio = {LORA_IRQ, LORA_IO1, LORA_IO2 },
 };
 
-static volatile transmitionStatus_t txStatus;
+static volatile transmissionStatus_t txStatus;
 static uint8_t * __rxPort;
 static uint8_t * __rxBuffer;
 static uint8_t * __rxSz;
 
+void event_log(const char *event){
+  log(DEBUG, "%s: %s", os_getTime(), event);
+}
 
-void onEvent (ev_t ev) {
-    Serial.print(os_getTime());
-    Serial.print(": ");
-    switch(ev) {
+void onEvent(ev_t ev) {
+    switch (ev) {
         case EV_SCAN_TIMEOUT:
-            Serial.println(F("EV_SCAN_TIMEOUT"));
             txStatus = TX_STATUS_ENDING_ERROR;
+            event_log(F("EV_SCAN_TIMEOUT"));
```

Comment:
You can (should) leave out the F(...)
on ESP32 ist is not necessary, ESP32 always puts const (literal) strings into flash and handles it transparently

Reply:
yeah, can clean that up later. for this PR will just keep it as it was before already.

Reply:
I didnt compile here with the F(
I removed them and it worked

Reply:
Oh? It compiled for me and it also compiled on travis-ci.

So is your setup somehow different?

Reply:
(hmm, ignore the "for me" case, I did not compile for lora hw)

but on travis, we compile for wifi and for lora hw.

Reply:
Here's the error message:

multigeiger/loraWan.cpp: In function 'void onEvent(ev_t)':
multigeiger/loraWan.cpp:133:43: error: cannot convert 'const __FlashStringHelper*' to 'const char*' for argument '1' to 'void event_log(const char*)'
             event_log(F("EV_SCAN_TIMEOUT"));

Reply:
OK, I configured for STICK and repective hw, now I also see that. Fixing...

Reply:
fixed.